themes/bootstrap3/templates/Recommend/SideFacets.phtml

```diff
@@ -4,6 +4,11 @@
   // Save results/options to $this so they are available to sub-templates:
   $this->results = $results = $this->recommend->getResults();
   $this->options = $options = $results->getOptions();
+
+  $hierarchicalFacets = $this->recommend->getHierarchicalFacets();
+  if ($hierarchicalFacets) {
+    $this->headScript()->appendFile('vendor/jsTree/jstree.min.js');
```

Comment:
Just curious why this is needed since it wasn't here before. Was something broken in the past, or is this necessary now due to some bit of refactoring I'm not noticing?

Reply:
jstree.min.js used to be injected by hierarchical-facet.js, but with deferred processing it's called too late to append anything to the headers. It could now be removed from hierarchical-facet.js but I kept it for back-compatibility.

Reply:
Thanks, that makes sense. I've added the first sentence of your reply as a comment for future reference.